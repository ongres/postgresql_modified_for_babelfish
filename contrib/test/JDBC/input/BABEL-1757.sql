CREATE TABLE t1(a int)
GO

SET FMTONLY ON SELECT * FROM t1 SET FMTONLY OFF
GO

DROP TABLE t1
GO

