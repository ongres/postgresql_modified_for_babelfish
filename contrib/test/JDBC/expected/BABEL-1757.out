CREATE TABLE t1(a int)
GO

SET FMTONLY ON SELECT * FROM t1 SET FMTONLY OFF
GO
~~START~~
int
~~END~~


DROP TABLE t1
GO

